Source/WebCore/loader/ImageLoader.cpp

```diff
@@ -598,7 +598,7 @@ void ImageLoader::dispatchPendingBeforeLoadEvent()
         return;
     if (!m_image)
         return;
-    if (!element().document().hasLivingRenderTree())
```

Comment:
hasLivingRenderTree is a trivial getter so we shouldn't need to do use protectedDocument here.

Reply:
Fixed.